rsign/signature.py

```diff
@@ -70,7 +72,10 @@ def compare(self, s1, s2):
     def sign_string(self, key, text):
         """ Return the signing method's digest """
         key, text = _clean(key), _clean(text)
-        return hmac.new(key, text, self.hash_fn).digest()
+        # Py3 hmac.new expects key as bytes. py2 expects str
```

Comment:
`_clean` function suppose to convert Unicode to bytes. It is not correctly implemented. I think it is better we fix `_clean` instead of adding the second conversion level.